src/wasm/kotlin/javatests/arcs/UnicodeTest.kt

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright 2019 Google LLC.
+ *
+ * This code may only be used under the BSD style license found at
+ * http://polymer.github.io/LICENSE.txt
+ *
+ * Code distributed by Google as part of this project is also subject to an additional IP rights
+ * grant found at
+ * http://polymer.github.io/PATENTS.txt
+ */
+
+package sdk.kotlin.javatests.arcs
```

Comment:
Something went wrong with the packaging here...  this should be `package arcs`

Reply:
All the test files in this dir are in that package (I cargo-culted one of the other tests for this file). Possibly due the to g3 import process?

Reply:
I think it was because the original author had something a little wrong with their AndroidStudio configuration.

Reply:
Think I'll leave that for another CL.

Reply:
Please don't.. it'll pollute the rest of the codebase with import statements using `sdk.kotlin.javatests`